documentation/assemblies/oauth/assembly-oauth.adoc

```diff
@@ -9,18 +9,19 @@
 
 {oauth} enables standardized token based authentication and authorization between applications, using a central authorization server to issue tokens that grant limited access to resources.
 
-{oauth} is currently only supported for authentication, with no authorization support.
+In {ProductName}, {oauth} is currently only supported for authentication, with no authorization support.
 However, {oauth} authentication can be used in conjunction with xref:simple-acl-using-uo[ACL-based Kafka authorization].
 
 Using {oauth} token based authentication, application clients can access resources on application servers (called ‘resource servers’) without exposing account credentials.
-The client passes an access token as a means of authenticating, which application servers can also use to find more information about the level of access granted.
+The client passes an access token as a means of authenticating, which application servers can also use to ascertain the level of access granted.
```

Comment:
Never saw `ascertain` before ... is there some workd more fitting into simplified English? ;-)

Reply:
I've changed to: 'to determine the level of access to grant.'